flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/BaseExpressions.java

```diff
@@ -1090,6 +1092,21 @@ public OutType regexpExtract(InType regex) {
                 unresolvedCall(REGEXP_EXTRACT, toExpr(), objectToExpression(regex)));
     }
 
+    /**
+     * Returns a string by quotes a string as a JSON value and wrapping it with double quote
+     * characters.
+     */
+    public OutType jsonQuote(InType input) {
+        return toApiSpecificExpression(
+                unresolvedCall(JSON_QUOTE, toExpr(), objectToExpression(input)));
+    }
+
+    /** Returns utf8mb4 string by unquoting JSON value. */
+    public OutType jsonUnquote(InType input) {
+        return toApiSpecificExpression(
+                unresolvedCall(JSON_UNQUOTE, toExpr(), objectToExpression(input)));
+    }
+
```

Comment:
How it is supposed to be called from table api?
can you provide a couple of examples?

Reply:
Good point. IT cases resolved some issues related to the table API and enabled examples